packages/http/README.md

```diff
@@ -1,3 +1,376 @@
-# Prism HTTP
+# Http Client
 
-TBD
+This package provides a http client featuring the ability to:
+
+- mock responses based of OAS spec
+- make requests to a http server
+- validate the request and/or the response according to the provided OAS
+
+The goal of this document is to provide you with some basic code examples to get you started and to cover some of the advanced scenarios.
+
+# Installation
+
+`yarn add @stoplight/prism-http`
+
+# Basic Usages
+
+> Note: examples in this document use the following spec
+
+```yaml
+---
+openapi: 3.0.2
+paths:
+  /todos:
+    get:
+      parameters:
+        - name: title
+          in: query
+          style: form
+          schema:
+            enum:
+              - eat
+              - drink
+      responses:
+        200:
+          description: Get Todo Items
+```
+
+## I want to mock responses of operations defined in an OAS file
+
+[Try it out!](https://repl.it/@ChrisMiaskowski/prism-http-client-basic-mocking)
```

Comment:
This is cool!

Reply:
Hey, I have built similar under personal account for Spectral. let's get a SL one and I will replace these at that point.